machine/dni/dni_3048up/INSTALL

```diff
@@ -0,0 +1,92 @@
+============================================================
+Installing ONIE on DNI 3048UP
+============================================================
+
+Cross-Compiling ONIE
+====================
+
+Change directories to ``build-config`` to compile ONIE.
+
+To compile ONIE first change directories to ``build-config`` and then
+type ``"make MACHINEROOT=../machine/dni/ MACHINE=dni_3048up all"``.
+For example::
+
+  $ cd build-config
+  $ make -j4 MACHINEROOT=../machine/dni/ MACHINE=dni_3048up all
+
+When complete, the ONIE binaries are located in
+``build/images``::
+
+  -rw-r--r-- 3604480 May  7 11:38 onie-dni_3048up-r1.bin
```

Comment:
I change the definition 'format' to uboot_ubootenv-up,
but the u-boot+env image range not fit mtd partition.
Because current script is:
===onie-mk-bin.sh===========================================
    pad_file=$(tempfile)
    dd if=/dev/zero of=$pad_file bs=$env_sector_size count=1 > /dev/null 2>&1 || {
        echo "ERROR: Problems with dd for $format image"
        exit 1
    }
    cat $UBOOT_BIN $pad_file > ${output_bin}.uboot+env
=========================================================

should i modify the script like that?
=========================================================
    pad_file=$(tempfile)
    dd if=/dev/zero of=$pad_file bs=$env_sector_size count=1 > /dev/null 2>&1 || {
        echo "ERROR: Problems with dd for $format image"
        exit 1
    }
    dd if=$UBOOT_BIN of=${output_bin}.uboot+env ibs=$uboot_max_size conv=sync > /dev/null 2>&1 || {
        echo "ERROR: Problems with dd for $format image"
        exit 1
    }
    cat $pad_file >> ${output_bin}.uboot+env
=========================================================